docs/.vitepress/buildEnd.config.ts

```diff
@@ -0,0 +1,49 @@
+import path from 'path'
+import { writeFileSync } from 'fs'
+import { Feed } from 'feed'
+import { createContentLoader, type SiteConfig } from 'vitepress'
+
+const siteUrl = 'https://vitejs.dev'
+const blogUrl = `${siteUrl}/blog`
+
+export const onBuildEnd = () => async (config: SiteConfig) => {
```

Comment:
Could you change this to be the direct hook? I don't think we need the extra indirection.
```suggestion
export const buildEnd = async (config: SiteConfig) => {
```

Reply:
Good point - at first I thought I'd need to pass some parameters from the main config but forgot yo clean it up.